enf/resource_enf_network.go

```diff
@@ -0,0 +1,95 @@
+package enf
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/xaptum/go-enf/enf"
+)
+
+func resourceEnfNetwork() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceEnfNetworkCreate,
+		Read:   resourceEnfNetworkRead,
+		Update: resourceEnfNetworkUpdate,
+		Delete: resourceEnfFirewallRuleDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
```

Comment:
Add support for importing an existing network into Terraform as well.  See the `Importer: &schema.ResourceImporter{` lines in `resource_enf_firewall_rule.go` for an example. 

Use the network id (<N>::/64) as the id for importing.  The firewall rule used an id that combined the network ID and the firewall rule ID (`<N>::/64.<UUID>`), so the importer `State` func is more complex.  This one should be more straightforward.

Reply:
I used the `schema.ImportStatePassthrough` function after reading through [this](https://www.terraform.io/docs/extend/resources/import.html) since the only property needed for the `Read` function is the resource ID (the network address).